docs/src/further_topics/filtering_warnings.rst

```diff
@@ -128,7 +128,9 @@ Or you can target Warnings raised by specific lines of specific modules, e.g.
     ...     warnings.filterwarnings("ignore", module="iris.coord_systems", lineno=449)
     ...     my_operation()
     ...
-    iris/coord_systems.py:800: IrisDefaultingWarning: Discarding false_easting and false_northing that are not used by Cartopy.
+    iris/coord_systems.py:434: IrisUserWarning: Setting inverse_flattening does not affect other properties of the GeogCS object. To change other properties set them explicitly or create a new GeogCS instance.
+      warnings.warn(wmsg, category=iris.exceptions.IrisUserWarning)
+    iris/coord_systems.py:772: IrisDefaultingWarning: Discarding false_easting and false_northing that are not used by Cartopy.
```

Comment:
Oops! Only just spotted this. The change should have instead been to update `lineno=` to correctly target the `IrisUserWarning`. #5736